HLTrigger/btau/plugins/HLTTauDxyFilter.cc

```diff
@@ -0,0 +1,81 @@
+#include "HLTTauDxyFilter.h"
+
+#include <vector>
+#include <string>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/RefToBase.h"
+#include "DataFormats/HLTReco/interface/TriggerFilterObjectWithRefs.h"
+#include "HLTrigger/HLTcore/interface/HLTFilter.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "HLTrigger/HLTcore/interface/defaultModuleLabel.h"
+
+
+//
+// constructors and destructor
+//
+
+HLTTauDxyFilter::HLTTauDxyFilter(const edm::ParameterSet& config)
+    : HLTFilter(config),
+      m_Taus(config.getParameter<edm::InputTag>("Taus")),
+      m_TausIP(config.getParameter<edm::InputTag>("TausIP")),
+      m_MinDxy(config.getParameter<double>("MinDxy")),
+      m_MaxDxy(config.getParameter<double>("MaxDxy")),
+      m_MinTaus(config.getParameter<int>("MinN")),
+      m_TriggerType(config.getParameter<int>("TriggerType")) {
+  m_TausToken = consumes<std::vector<reco::PFTau>>(m_Taus), 
+  m_TausIPToken = consumes<edm::AssociationVector<reco::PFTauRefProd,std::vector<reco::PFTauTransverseImpactParameterRef>>>(m_TausIP),
+
+  edm::LogInfo("") << " (HLTTauDxyFilter) trigger cuts: " << std::endl
+                   << "\tmin/max dxy value: [" << m_MinDxy << ".." << m_MaxDxy << "]" << std::endl
+                   << "\tmin no. passing taus: " << m_MinTaus << "\tTriggerType: " << m_TriggerType << std::endl;
```

Comment:
```suggestion
                   << "\tmin no. passing taus: " << m_MinTaus << "\tTriggerType: " << m_TriggerType;
```
I might be misremembering, but I dont think endl has any effect at the end of `LogInfo` (it makes a new line anyway).